src/ruby/lib/symengine.rb

```diff
@@ -0,0 +1,12 @@
+require 'symengine/symengine'
+
+module SymEngine
+  def SymEngine.ascii_art
+    puts "
+ _____           _____         _         
+|   __|_ _ _____|   __|___ ___|_|___ ___ 
+|__   | | |     |   __|   | . | |   | -_|
+|_____|_  |_|_|_|_____|_|_|_  |_|_|_|___|
+      |___|               |___|          "
+  end
+end
```

Comment:
As to this ascii art, I think that's a great idea.

I played with other options, here is a list of various fonts: https://gist.github.com/certik/a78cef8105f0c4b34e41, the one you used is:
```
Font: rectangles
                                         
 _____           _____         _         
|   __|_ _ _____|   __|___ ___|_|___ ___ 
|__   | | |     |   __|   | . | |   | -_|
|_____|_  |_|_|_|_____|_|_|_  |_|_|_|___|
      |___|               |___|          
```
Here are other options that I found pretty readable as well from the list:
```
Font: big
  _____                 ______             _            
 / ____|               |  ____|           (_)           
| (___  _   _ _ __ ___ | |__   _ __   __ _ _ _ __   ___ 
 \___ \| | | | '_ ` _ \|  __| | '_ \ / _` | | '_ \ / _ \
 ____) | |_| | | | | | | |____| | | | (_| | | | | |  __/
|_____/ \__, |_| |_| |_|______|_| |_|\__, |_|_| |_|\___|
         __/ |                        __/ |             
        |___/                        |___/              

Font: doom
 _____                 _____            _            
/  ___|               |  ___|          (_)           
\ `--. _   _ _ __ ___ | |__ _ __   __ _ _ _ __   ___ 
 `--. \ | | | '_ ` _ \|  __| '_ \ / _` | | '_ \ / _ \
/\__/ / |_| | | | | | | |__| | | | (_| | | | | |  __/
\____/ \__, |_| |_| |_\____/_| |_|\__, |_|_| |_|\___|
        __/ |                      __/ |             
       |___/                      |___/              

Font: small
 ___            ___           _          
/ __|_  _ _ __ | __|_ _  __ _(_)_ _  ___ 
\__ \ || | '  \| _|| ' \/ _` | | ' \/ -_)
|___/\_, |_|_|_|___|_||_\__, |_|_||_\___|
     |__/               |___/            

Font: standard
 ____                  _____             _            
/ ___| _   _ _ __ ___ | ____|_ __   __ _(_)_ __   ___ 
\___ \| | | | '_ ` _ \|  _| | '_ \ / _` | | '_ \ / _ \
 ___) | |_| | | | | | | |___| | | | (_| | | | | |  __/
|____/ \__, |_| |_| |_|_____|_| |_|\__, |_|_| |_|\___|
       |___/                       |___/              

Font: ogre
 __                   __            _            
/ _\_   _ _ __ ___   /__\ __   __ _(_)_ __   ___ 
\ \| | | | '_ ` _ \ /_\| '_ \ / _` | | '_ \ / _ \
_\ \ |_| | | | | | //__| | | | (_| | | | | |  __/
\__/\__, |_| |_| |_\__/|_| |_|\__, |_|_| |_|\___|
    |___/                     |___/              
```
Maybe some of them look better. What do you think?